src/ui/css/vars.scss

```diff
@@ -53,9 +53,13 @@ $font-size-header-title: 34px;
 
 // Border radius sizes
 $border-radius-default: 9px;
-$border-radius-s: 4px;
-$border-radius-m: 8px;
+$border-radius-s: 3px;
```

Comment:
I think it's dangerous that we have variables that are named the same as those in https://github.com/mozilla/addons-frontend/blob/67da4111618ba06c0967e461929174597aeba7e6/src/core/css/inc/vars.scss#L35-L37

...because Sass won't even tell you when an import overwrites variables from a previous import. Could we call these `$ui-border-radius-`, maybe?

Also, why change `$border-radius-s` to 3px? Was that only for buttons? My concern is that we use `$border-radius-s` all over the place and the Sketch file still has it at 4px for most things. But, again, maybe changing the variable names would address my concern.

Reply:
I'd like to change the variable names; that's a good point. Changing the size though was seemingly unrelated and I've undone it. But I think it's worth doing in another patch as this is unrelated 😄